aesthetic_ants/penguin/main.py

```diff
@@ -1,11 +1,27 @@
 import pyglet
+from enemy import Enemy
 
 window = pyglet.window.Window(caption='Penguin Snowball')
+pyglet.resource.path = ['../resources']
+pyglet.resource.reindex()
+enemy_image = pyglet.resource.image('circle.png')
+enemy_image.width = 50
+enemy_image.height = 50
+
+
+enemy = Enemy(enemy_image)
 
```

Comment:
Ideally you'd keep the enemy_image in the Enemy's `__init__` as they're not really relevant to the main loop.

This also saves you from having to pass the image explicitly, completely up to you though.